mmdet/models/roi_heads/cascade_roi_head.py

```diff
@@ -58,13 +58,11 @@ def init_bbox_head(self, bbox_roi_extractor, bbox_head):
         """
         self.bbox_roi_extractor = ModuleList()
         self.bbox_head = ModuleList()
+        count = len(bbox_head)
```

Comment:
The name is not good, maybe you can consider `num_bbox_heads`, or other

Reply:
done! :smile: